tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3011,7 +3011,10 @@ public function adminCreatesGroupUsingTheProvisioningApi(string $group):void {
 	 */
 	public function groupHasBeenCreated(string $group):void {
 		$this->createTheGroup($group);
-		$this->groupShouldExist($group);
+		Assert::assertTrue(
+			$this->groupExists($group),
+			"Group '$group' should exist but does not exist"
+		);
```

Comment:
This should be fine, why remove it and repeat the exact line that is inside the function `groupShouldExist`?
```suggestion
		$this->groupShouldExist($group)
```

Reply:
Yes! you are correct.